ebundala/free-mentors
# #168148202 Create open Api specification

#### What does this PR do?
add open api specification 
#### Description of Task to be completed?
create openapi,yaml file for specs
host the swagger docs on the app web address
#### How should this be manually tested?
to test visit this endpoint on top a base address
 /api/v1/api-docs 
then you shuld see the swaggger documentation for the api
#### What are the relevant pivotal tracker stories?
[#168148202](https://www.pivotaltracker.com/story/show/168148202)
#### Screenshots 
![localhost_3008_api_v1_api-docs_](https://user-images.githubusercontent.com/19614438/63851051-1ff9ef80-c99e-11e9-8649-0ef157065653.png)
